### ISSUE TITLE
Evaluation results for test set

### ISSUE THREAD

**Issue by praveen5733** [Isalia20/DETR-finetune#1]
Hi, 

Thanks for this work. In your results section, you mention 59.0 mAP on the SKU110K validation set. Do you have the numbers for the test set?

I appreciate any help you can provide.

Praveen

**Comment by Isalia20** [comment]
I don't have numbers for the test set however models are easily loadable with huggingface(transformers library) interface. Here are the models(with the best checkpoint), you can get the metrics with these:
https://huggingface.co/isalia99/detr-resnet-50-sku110k
https://huggingface.co/isalia99/detr-resnet-101-dc5-sku110k

Let me know if you need any help with loading the models. I haven't done much hyperparameter tuning due to having limited compute so I don't expect the number to be too different on the test set.
